aries-site/src/pages/learn/how-to-add-search-and-filter-to-datatable-with-data.mdx

```diff
@@ -0,0 +1,196 @@
+import { Box, Image, Notification } from 'grommet';
+
+## Prerequisites
+
+Let’s get started!
+
+Use this [CodeSandbox template](https://codesandbox.io/s/how-to-add-search-and-filter-to-datatable-with-data-7gxmcy?file=/index.js) to start. The template is populated with a paginated DataTable configured to display SpaceX data.
+
+<Notification
+  status="info"
+  width="large"
+  margin={{ bottom: 'medium' }}
+  message="If you’d like to learn more about how to configure a DataTable with a REST API, check out the guide."
+  actions={[
+    {
+      label: 'Guide TBD',
+      href: '#',
+    },
+  ]}
+/>
+
+## Set up Data provider
+
+In `index.js`, start by importing `Data` from grommet.
+
+Then, remove the `data` property from DataTable and place it as a property on Data. The Data component is now the data provider, controlling which data should be rendered in the DataTable.
+
+```
+import { Box, Data, DataTable, Grommet, Pagination, Text } from "grommet";
+
+...
+
+<Data data={data}>
+    <DataTable columns={columns} />
+</Data>
+```
+
+## Add default search and filter capabilities
+
+### Toolbar
+
+Add `toolbar` as a boolean property on Data to display a search input, filter button, and results summary above the DataTable.
+
+```
+<Data data={data} toolbar>
+```
+
+At this point, your project should look like this:
+
+<Box
+  width="medium"
+  round="xsmall"
+  overflow="auto"
+  margin={{ bottom: 'medium' }}
+>
+  <Image
+    src="/learnImages/data-simple-toolbar.png"
+    alt="Toolbar containing a search input and filter button located above a DataTable displaying SpaceX launch data."
+    fit="contain"
+  />
+</Box>
+
+### Total
+
+We are only fetching one page of data at a time. Add `total={numberItems}` to inform Data of the total number of records.
+
+`numberItems` is defined in the starter template using a key from the API response providing the total number of records in the table.
+
+```
+<Data data={data} total={numberItems} toolbar>
+    <DataTable columns={columns} />
+</Data>
+```
+
+### View and onView
+
+The `view` property specifies the subset of data to be displayed, whether that be applied filters, current page, and/or other customizations.
+
+To ensure the table displays the proper results from any applied filters, use `view` and `onView`.
+
+First, create a state variable called `view`.
```

Comment:
are we assuming they know react `state` should we link to react docs or just assume our users should know

Reply:
For this recipe, I'm assuming they have basic react knowledge

Reply:
fair